package-lock.json

```diff
@@ -2801,7 +2801,7 @@
         "lodash": "^4.17.11",
         "oboe": "2.1.4",
         "url-parse": "1.4.4",
-        "websocket": "git://github.com/frozeman/WebSocket-Node.git#browserifyCompatible",
+        "websocket": "git://github.com/frozeman/WebSocket-Node.git#6c72925e3f8aaaea8dc8450f97627e85263999f2",
```

Comment:
`npm` reverts it back. Why is this manual change needed?

Reply:
Please never edit `package-lock.json` - it is ofter regenerated by `npm`.

Reply:
@varasev, do we need `package-lock.json` to be version-controlled at all? When `npm i` runs, it changes it quite a bit. I'm not sure what difference it makes to have it there in the first place.

Reply:
Yes, it should be in the repo: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
Every time `npm i` is run as part of `npm run all`, this file changes, sometimes dramatically. Therefore I'm not sure about deterministic installs.